src/locale/en.ts

```diff
@@ -503,6 +503,7 @@ I18n.translations.en = {
         },
         table: {
             jira_ticket_id: "Jira ticket",
+            ims_ticket_id: "IMS Planned work",
```

Comment:
Is it possible to name this `ims_pw_id` as well for consistency?

Reply:
Yes I can.
Note that it's very inconsistent form an IMS perspective. All other IMS id fields are int's: and are actual IMS DB ID's

This one is not: so made it consistent for the user here. But I'm also fine to make the translation consistent with the payload keys.